src/osm_configurator/view/states/state.py

```diff
@@ -27,53 +33,79 @@ def __init__(self, active_frames, own_state_name, default_left, default_right):
             default_left (state_name_enum.StateName): The name of the state on this states left
             default_right (state_name_enum.StateName): The name of the state on this states right
         """
-        pass
 
-    def get_active_frames(self):
+        # Checking if active_frames are a list, and all of its elements are PositionedFrames
+        if not isinstance(active_frames, list):
```

Comment:
this isinstanceof isn't really THAT bad, but i feel like it really maks the code less readable

Reply:
I keep my checks for correct types